app/models/gogovan_order.rb

```diff
@@ -24,6 +26,12 @@ def update_status(status)
 
   private
 
+  def self.set_vehicle_type(attributes)
+    offer = Offer.find(attributes['offerId'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.